lib/alephant/logger/json.rb

```diff
@@ -37,10 +39,16 @@ def session?
 
       private
 
+      attr_reader :desired_level
+
       def write(hash)
         @log_file.write(::JSON.generate(hash) + "\n")
       end
 
+      def write_level?(level)
+        Alephant::Logger::Level.new(desired_level, level).log?
+      end
```

Comment:
I wonder if the names of the parameters here (`desired_level`, `level`) could be clearer in their purpose.

Alternately what do you think about changing the interface so that instead of a two-arg constructor there is a one-arg constructor and a method, e.g. something like:

```
Alephant::Logger::Level.new(log_level).logs?(message_level)
```
?